TimberAPI/EntityInstantiator/EntityInstantiator.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using HarmonyLib;
+using Timberborn.EntitySystem;
+using UnityEngine;
+
+namespace TimberbornAPI.EntityInstantiatorSystem
+{
+    internal class EntityInstantiator
```

Comment:
For me an `EntityInstantiator` would be a class that creates entity instances. But this class applies actions to `GameObject`s *when* they're instantiated.
How about `GameObjectActionApplier` for this class? Or something more catchy like `GameObjectTweaker`?
And consequently:
`TimberbornAPI.EntityInstantiatorSystem` -> `TimberbornAPI.GameObjectActions` or `TimberbornAPI.GameObjectTweaking`?
`IEntityInstantiator` -> `IGameObjectAction`?
`_gameObjectModifiers` -> `_instantiationActions`?
`visualElementInitializers` -> `instantiationActions`?